overlay.yaml

```diff
@@ -2,7 +2,14 @@ overlay: 1.0.0
 info:
   title: Overlay prod-openapi.yaml => openapinew.yaml
   version: 0.0.0
+  x-speakeasy-jsonpath: rfc9535
```

Comment:
oh I think this is supposed to go above the info, so maybe that's why what I sent you wasn't working for you